Library/Formula/vera++.rb

```diff
@@ -2,26 +2,21 @@
 
 class Veraxx < Formula
   homepage "https://bitbucket.org/verateam/vera"
-  url "https://bitbucket.org/verateam/vera/downloads/vera++-1.2.1.tar.gz"
-  sha1 "0bf7d463dabe41c2069dbe2fa4f7fca192cb7d6e"
+  url "https://bitbucket.org/verateam/vera/downloads/vera++-1.3.0.tar.gz"
+  sha1 "696c143e42df6854c9e5674b9e713fa8b9dae6f0"
 
-  depends_on "cmake" => :build
-  depends_on "boost"
-
-  # Use prebuilt docs to avoid need for pandoc
-  resource "doc" do
-    url "https://bitbucket.org/verateam/vera/downloads/vera++-1.2.1-doc.tar.gz"
-    sha1 "fce30676c815b99aa66d25c9dfbd2eda2c74bd7a"
+  bottle do
   end
 
+  depends_on "cmake" => :build
+  depends_on "pandoc" => :build
+  depends_on "lua51"
```

Comment:
vera++ builds fine with lua 5.2, but luabind is linked to lua 5.1.

There is an option in vera++ to download lua 5.2 and a dev version of luabind, and linked them statically to the vera++ executable. Would that be better than linking to lua 5.1 and luabind libs?